content/applications/essentials/contacts.rst

```diff
@@ -178,13 +178,64 @@ section, and other partner options, including :guilabel:`Partner Activation` and
 Review` configurations. These are **only** present when the *Enterprise Resellers* module is
 installed.
 
+To publish a list of all active partners on the website, a new webpage must be created first.
+Navigate to :menuselection:`Website app --> Site --> Pages` and click :guilabel:`New`. Hover over
+the :guilabel:`Blank Page` template, and click the :guilabel:`Use this template` button to create a
+new page.
```

Comment:
```suggestion
To publish a list of all active partners on the website, a new webpage must be created first.

To create a new webpage, navigate to :menuselection:`Website app --> Site --> Pages`, and click :guilabel:`New`. Hover over
the :guilabel:`Blank Page` template, and click the :guilabel:`Use this template` button to create a
new page.
```
SUGGESTION: broke up the paragraph, added comma for enhanced sentence flow/pacing

This is a totally viable way to create a new page. However, I think the same can be accomplished by going to Website app --> Clicking +New in the upper-right corner --> Selecting 'Page' from the presented options --> and selecting the 'Blank Page' template as you've instructed above.

Just something to consider adding during your revisions.